### PR TITLE
TT-5628 Trying to find file online while online...couldn't reproduce to just guessing

### PR DESCRIPTION
**Pull request by sarahentzel**

### CODE REVIEW

File: src/crud/useFetchMediaUrl.ts [hunk 1]
```diff
@@ -125,21 +126,22 @@ export const useFetchMediaUrl = (reporter?: any) => {
...
             const audioUrl =
               mediarec.attributes.audioUrl ??
               mediarec.attributes.s3file ??
               mediarec.attributes.originalFile;
-            const path = await dataPath(audioUrl, PathType.MEDIA);
-            const foundLocal = await ipc?.exists(path);
-            if (foundLocal || !accessToken) {
+            const path = await dataPath(audioUrl, PathType.MEDIA, local);
+            const foundLocal = local.localname === path;
+            if (foundLocal || offline) {
```
Comment (gtryus) [inline src/crud/useFetchMediaUrl.ts:136]: line 136 says if foundlocal or offline so if offline is tru it will never get to 144... Did you mean && (and)?

Comment (sarahentzel) [reply src/crud/useFetchMediaUrl.ts:136]: 144 is inside the if of 136

File: src/crud/useFetchMediaUrl.ts [hunk 2]
```diff
@@ -125,21 +126,22 @@ export const useFetchMediaUrl = (reporter?: any) => {
...
                 if (cancelled()) return;
                 dispatch({
                   payload: await safeURL(path),
                   type: MediaSt.FETCHED,
                 });
                 return;
-              } else if (!accessToken) {
+              } else if (offline) {
                 dispatch({
                   payload: 'no offline file',
```
Comment (gtryus) [inline src/crud/useFetchMediaUrl.ts:146]: This should be a localized error

Comment (sarahentzel) [reply src/crud/useFetchMediaUrl.ts:146]: It's localized by the receiver of the media_state

Comment (sarahentzel) [reply src/crud/useFetchMediaUrl.ts:146]: if (mediaState.error.startsWith('no offline file'))
        showMessage(sharedStr.fileNotFound);

**Pull request merged**